spec/model/force/application_spec.rb

```diff
@@ -3,14 +3,16 @@
 require 'rails_helper'
 
 RSpec.describe Force::Application do
+  let(:paper_app_custom_api) { fixture('model/force/application/paper_app_custom_api.json') }
```

Comment:
I'd like to add even more tests here to make sure the functions are really well covered. We have had so many bugs in the past becuase of missing fields or things being mis-translated. This is our chance to be really explicit about what we want to happen when we map from domain to customAPI